openapi.yaml

```diff
@@ -582,7 +582,7 @@ definitions:
         format: signedCheckpoint
         description: The current signed tree head
       treeID:
-        type: integer
+        type: string
```

Comment:
should we add a pattern here to help protect against bad input, e.g:

`pattern: '^[0-9]+'`

Reply:
sgtm, updated to '^[0-9]+$'